spec/jobs/friendship_article_job_spec.rb

```diff
@@ -0,0 +1,53 @@
+RSpec.describe FriendshipArticleJob, type: :job do
+  let(:user) { create :user }
+  let(:friend) { create :user, first_name: "Иван" }
+  let(:action) { nil }
+
+  subject(:perform_job) { described_class.perform_now(action, user.id, friend.id) }
+
+  describe "#perform" do
+    context "when action is :request_sent" do
+      let(:action) { :request_sent }
+
+      it "calls Articles::Create with correct parameters" do
+        expect(Articles::Create).to receive(:call).with(
+          user: user,
+          params: {
+            title: "Заявка отправлена",
+            body: "Я отправил заявку Иван",
+            status: "public"
+          }
+        )
+
+        perform_job
```

Comment:
А так сработает? Проверка что реально вызвался класс после выполнения джобы

```suggestion
        expect(Articles::Create).to receive(:call).with(
          user: user,
          params: {
            title: "Заявка отправлена",
            body: "Я отправил заявку Иван",
            status: "public"
          }
        )

        perform_job
        
       expect(Articles::Create).to have_received(:call)
```